docs/source/reference/image_embedder.rst

```diff
@@ -0,0 +1,154 @@
+
+.. _image_embedder:
+
+##############
+Image Embedder
+##############
+
+********
+The task
+********
+Image embedding encodes an image into a vector of image features which can be used for anything like clustering, similarity
+search or classification.
+
+------
+
+*********
+Inference
+*********
+
+The :class:`~flash.vision.ImageEmbedder` is already pre-trained on [ImageNet](http://www.image-net.org/), a dataset of over 14 million images.
+
+Use the :class:`~flash.text.ImageEmbedder` pretrained model for inference on any image tensor or image path using :func:`~flash.text.ImageEmbedder.predict`:
+
+.. code-block:: python
```

Comment:
could it run on CPU? I would consider turning it to `.. testcode::` so we are on the safe side that it is working